view/frontend/templates/js/replacejs.phtml

```diff
@@ -1707,7 +1707,7 @@ if (!$block->isSaveCartInSections()) { ?>
         var cartBarrier;
         var countConfigureCalls = 0;
 
-        var boltCheckoutConfigure = function(cart, hints, callback, parameters = undefined) {
+        var boltCheckoutConfigure = function(cart, hints, callback, parameters) {
```

Comment:
Is there not any ill effect of no longer having a default set here?

Reply:
I think that no effects.
in JS If a function is called with missing arguments (less than declared), the missing values are set to: undefined.
So if we didn't pass **parameters**, it set to undefined.
Actually **parameters = undefined** makes no sense.